libsolidity/analysis/TypeChecker.cpp

```diff
@@ -1807,26 +1802,15 @@ void TypeChecker::requireLValue(Expression const& _expression)
 
 void TypeChecker::typeError(SourceLocation const& _location, string const& _description)
```

Comment:
I assume in a subsequent PR we could think about replacing these with direct calls.